build/build.xml

```
@@ -3,6 +3,21 @@
   xmlns:if="ant:if" xmlns:unless="ant:unless">
   <description>
     Perform common search an replace tasks to support the release process.
+    
+    Information on the current build environment:
+       ant info
+    
+    Update code references during release:
+       ant release -Drelease='27.0'
+       
+    Update main to a new snapshot version:
+       ant latest -Drelease='28-SNAPSHOT
```

Does this need a closing quote?

So:
```suggestion
       ant latest -Drelease='28-SNAPSHOT'
```